tests/cpp/CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ file(GLOB tests_src "*.cpp")
 
 set(TEST_TARGET_NAME "tests_continuous_batching")
 
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
I suppose it should be right after `set(INSTALL_GTEST OFF CACHE BOOL "")` otherwise, `FetchContent_MakeAvailable` will not respect this value

Reply:
reverted